plugins/afl-scoreboard/game_renderer.py

```
@@ -0,0 +1,550 @@
+                bbox = logo.getbbox()
+                if bbox:
+                    logo = logo.crop(bbox)
+                logo.thumbnail((self.display_height, self.display_height), Image.Resampling.LANCZOS)
+
+                self._logo_cache[team_abbrev] = logo
+                return logo
+            else:
+                self.logger.debug(f"Logo not found at {logo_path}")
+                return None
+                
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)